### DISCUSSION TITLE
Detecting pauses and removing them

### DISCUSSION THREAD

**Discussion by niteshsharoff** [Zulko/moviepy#1829] [Q&A]
I'm keen to write code to do these things:
- Detect pauses in audio and remove that block
- Also zoom the video in ever so slightly (110%)
- And in the following pause, zoom back out (100%)

Any ideas how to do this using this library?

**Comment by keikoro** [comment]
Please open a proper issue for this (using our issue templates) instead of using GitHub's Discussions feature, thank you.
